libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -610,6 +610,16 @@ AP_GPS::first_unconfigured_gps(void) const
 }
 
 void
+AP_GPS::broadcast_first_configuration_failure_reason(void) const {
+    uint8_t unconfigured = first_unconfigured_gps();
+    if(drivers[unconfigured] == NULL) {
+        GCS_MAVLINK::send_statustext_all(MAV_SEVERITY_INFO, "GPS %d: was not found", unconfigured + 1);
```

Comment:
The default configuration only looks for one GPS, the only time a second GPS is enabled is when the user goes out of their way to do so. This is the only way to automatically detect that the a second GPS can't actually be found at the moment.